test/integration/scheduler/taint_test.go

```diff
@@ -656,6 +656,8 @@ func TestTaintBasedEvictions(t *testing.T) {
 	for i, test := range tests {
 		t.Run(test.name, func(t *testing.T) {
 			context := initTestMaster(t, "taint-based-evictions", admission)
+			defer context.closeFn()
```

Comment:
unrelated?

Reply:
> unrelated?

found it working on this one, just a forgotten close